docs/architecture.md

```diff
@@ -8,209 +8,175 @@
 
 Taking into account that [web3.storage](http://web3.storage) onboards any type of content (up to a maximum of 4GiB-padded shards to have better utilization of Fil sector space), multiple CAR files uploaded need to be aggregated into a bigger Piece that can be offered to Filecoin Storage Providers. w3filecoin pipeline keeps track of queued CARs (cargo) to be included in Storage Provider deals.
 
-When a CAR file is written into a given web3.storage's bucket, its metadata gets into the w3filecoin processing pipeline. This pipeline is composed of multiple processing queues, together with a job scheduler per queue that will perform the processing. Each queue handles a processing stage with the goal of getting CAR files into Filecoin deals with Storage Providers.
+When a CAR file is written into a given web3.storage's bucket, its piece is computed and sent into the w3filecoin processing pipeline. This pipeline is composed of multiple processing queues that will perform the processing of pieces until they land into a Filecoin deal.
 
 ## High Level design
 
 The high level flow for the w3filecoin Pipeline is:
 
-- **Consume request** is received once a CAR file is written into a bucket with its metadata {`link`, `size`, `bucketName`, `bucketUrl`, `bucketRegion`, `key`}. This event is added to a `content_validator_queue`.
-- **Content Validator process** validates CARs and writes their references to a `content` table.
-- On its own schedule, **Piece maker process** can pull queued content from a `content_queue`, derive piece info for them and write records to a `piece` & `inclusion` tables.
-  - a `inclusion` table enables same piece to be included in another aggregate if a deal fails.
-- **Agregagtor process** reads from a `cargo_queue` (backed by `inclusion` table), attempts to create an aggregate and if successful it writes to an `aggregate` table.
-- **Submission process** reads from the `aggregate_queue`, submits aggregates to the agency (spade proxy) and writes deal record with status "PENDING".
-- TBD deal flow
+- **piece inclusion request** is received by a Storefront with `pieceCid` (TBD `endpoint` to get presigned urls from given we want to support w3up, web3.storage, ...?)
+- **piece queued** to be aggregated
+- **piece contenation** into ferries until a ferry has enough load to become an aggregate offer
+- **aggregate submission** to Storage Provider broker
+- **deal tracking** and **deal recording** once fulfilled
+
+TODO: Update diagram
 
 ![Pipeline processes](./processes.svg)
 
 The w3filecoin pipeline is modeled into 3 different SST Stacks that will have their infrastructure provisioned in AWS via AWS CloudFormation. These are:
 
 - API Stack
-- DB Stack
 - Processor Stack
+- Data Stack
+
+TODO: Update diagram
```

Comment:
Created this version
https://www.tldraw.com/s/v2_c_mT6CNk8MHTN8D7zNIHKPu?viewport=-433%2C-168%2C2137%2C1618&page=page%3ANcTbTvpauUBwQxgzeo96z

![w3-aggregation](https://github.com/web3-storage/w3filecoin/assets/21236/ccbe7663-ced2-43e4-a78a-102fbf3462a1)